atomics/T1110.003/T1110.003.yaml

```diff
@@ -105,3 +105,59 @@ atomic_tests:
         }
       }
       Write-Host "End of password spraying"
+
+- name: Password spray all Azure AD users with a single password 
+  auto_generated_guid: a8aa2d3e-1c52-4016-bc73-0f8854cfa80a
+  description: |
+    Attempt to brute force all Azure AD users with a single password (called "password spraying") via AzureAD Powershell module.
+    Valid credentials are only needed to fetch the list of Azure AD users.
+  supported_platforms:
+  - azure-ad
+  input_arguments:
+    password:
+      description: Single password we will attempt to auth with (if you need several passwords, then it is a bruteforce so see T1110.001)
+      type: String
+      default: P@ssw0rd!
+    valid_username:
+      description: Valid username to retrieve Azure AD users
+      type: String
+      default: bruce.wayne@contoso.com
+    valid_password:
+      description: Valid password to authenticate as valid_username
+      type: string
+      default: iamthebatman
```

Comment:
And also, a valid_test_password as this particular atomic attempts to log using `valid_username` and `valid_password` to fetch other users

Other than that, this test worked right away! Thank you

Reply:
Hello ! I don't fully understand what you mean. Do you wish to rename parameters like valid_password -> valid_test_password and valid_username -> valid_test_username ? Or just highlight more in the description that it must be a valid combo username/password ?

Reply:
Hey! sorry for being unclear. I meant the latter; highlight more in the description that it must be a valid combo username/password